setup.py

```diff
@@ -65,6 +65,11 @@ def rtd_dependent_deps():
     ]
     + rtd_dependent_deps(),
     extras_require=extra_require,
-    entry_points={"console_scripts": ["manticore = manticore.__main__:main"]},
+    entry_points={
+        "console_scripts": [
+            "manticore = manticore.__main__:main",
+            "manticore_verifier = manticore.ethereum.verifier:main",
```

Comment:
We have a few integration tests that call `manticore` via subprocess, which in effect confirm that the console script install was successful. Let's add something similar for `manticore_verifier`, since it looks like a lot of the uncovered code is in the argument parsing and setup for the verifier CLI target.

Reply:
https://github.com/trailofbits/manticore/pull/1717/files#diff-9a2242beaf0823573028164bdf320ba7R89-R94